catalog/app/components/JsonDisplay/JsonDisplay.tsx

```diff
@@ -66,14 +66,15 @@ const useStyles = M.makeStyles((t) => ({
   iconBlank: {
     paddingRight: t.spacing(2.5),
   },
-  key: {
-    fontWeight: t.typography.fontWeightBold,
+  key: {},
+  value: {
+    opacity: 0.7,
```

Comment:
```suggestion
    color: t.palette.text.secondary
```

Reply:
Visually similar

Reply:
`.separator` and `.hint` opacity rules can also be replaced with `t.palette.text.hint`. Or, was your intention to make component independent of color scheme?

Reply:
> Or, was your intention to make component independent of color scheme?

yep. since we don't have hardcoded colors there, i've decided to keep it that way and just use opacity